test/parallel/test-tls-connect-given-socket.js

```diff
@@ -72,14 +66,14 @@ const server = tls.createServer(options, (socket) => {
 
   // Immediate death socket
   const immediateDeath = net.connect(port);
-  establish(immediateDeath).destroy();
+  establish(immediateDeath, 0).destroy();
```

Comment:
This one was a bit of let's get it working. `establish()` is called multiple times with different expected results each time and this was the easiest to get things going. The test as written, however, could use a bit of a bigger refactoring.